.github/workflows/build_test.yml

```
@@ -0,0 +1,60 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Build & test
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
```

If we filter nothing, we might as well skip `branches`
```suggestion
on: [push, pull_request]
```
or even
```suggestion
on: pull_request
```